packages/net/writebuffer.pony

```diff
@@ -0,0 +1,155 @@
+use "collections"
+
+class WriteBuffer
+  """
+  * A List[ByteSeq] maintained as WriteBuffer.
```

Comment:
I dont understand from this documentation how I'm supposed to use this class.

When would I want to use it? Why? How do I go about using it?

What's an MT() in the example? 
Why does the WBUDPNotify start writing to a socket when it starts listening?

I'm left with a ton of questions.